tests/llmcompressor/pipelines/layerwise/test_helpers.py

```
@@ -0,0 +1,295 @@
+            weight_map = build_weight_map(str(tmpdir))
+            param_names = set(name for name, _ in qwen3_model.named_parameters())
+            assert set(weight_map.keys()) == param_names
+
+
+class TestBuildKeyRemapping:
+    """Tests for build_key_remapping."""
+
+    def test_no_remapping_needed(self, qwen3_model, fake_safetensors_dir):
+        """When model params match safetensors keys, no remapping should occur."""
+        weight_map = build_weight_map(str(fake_safetensors_dir))
+        remapped_wm, model_to_sf, passthrough, tied = build_key_remapping(
```

Ruff (0.15.13)

[warning] Unpacked variable `remapped_wm` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)